packages/loopring_v3/contracts/core/impl/libexchange/ExchangeWithdrawals.sol

```diff
@@ -480,31 +480,35 @@ library ExchangeWithdrawals
         private
         returns (bool success)
     {
-        if (nft.token == nft.minter) {
-            // This is an existing thirdparty NFT contract
-            success = ExchangeNFT.withdraw(
-                S,
-                from,
-                to,
-                nft.nftType,
-                nft.token,
-                nft.nftID,
-                amount,
-                extraData,
-                gasLimit
-            );
+        if (gasLimit > 0) {
```

Comment:
maybe cleaner as:
```solidity
if (gasLimit == 0) return false;
...

Reply:
The function shouldn't immediately return because the checks + event at the end of the function should still be done (why I tend to not do early outs often in functions and just do if/else).